package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "linter-gotype",
   "linter-package": true,
-  "main": "./lib/init",
+  "main": "./lib/linter-gotype",
```

```suggestion
  "main": "./lib/init",
```

Can you actually rename the main file to `init.js`? It makes no functional difference, but it lets the `package.json` and the require in the specs look more similar between all the packages.